pom.xml

```diff
@@ -51,23 +51,23 @@
     </distributionManagement>
 
     <properties>
-        <keycloak.version>12.0.4</keycloak.version>
+        <keycloak.version>15.0.2</keycloak.version>
```

Comment:
Is this the required keycloak version now? If so, it should be documentated.

Reply:
Hm, I'm not sure if keycloak is required in that version, too. I assumed that you can access an older version of keycloak with a newer version of the lib as well, similar to a newer jdbc lib version. Is that not so?

Reply:
In general I would prefer to have a separate build for any given keycloak version.

But actually I think this change should work as the update didn't seem to introduce any breaking changes in our plugin.